sign-and-combine-ecdsa/nodejs/project.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "sign-and-combine-ecdsa-nodejs",
+    "root": "sign-and-combine-ecdsa/nodejs",
+    "targets": {
+        "test-lit": {
+            "executor": "nx:run-commands",
+            "options": {
+                "command": "cd sign-and-combine-ecdsa/nodejs && yarn && yarn test"
+            }
+        },
+        "update-lit": {
+            "executor": "nx:run-commands",
+            "options": {
+                "command": "cd sign-and-combine-ecdsa/nodejs && chmod +x ../../scripts/update-lit.sh && ../../scripts/update-lit.sh"
```

Comment:
is there a reason we need to make the file executable each time we run the project commands (apples to all project.json implementations).

Reply:
It was needed for me initially when I created the script but it's not needed multiple times now, thankyou for pointing this out, I'll update this